content/en/docs/setup/getting-started/index.md

```diff
@@ -388,13 +388,20 @@ We welcome you to ask questions and give us feedback by joining the
 
 ## Uninstall
 
-The uninstall deletes the RBAC permissions, the `istio-system` namespace, and
-all resources hierarchically under it. It is safe to ignore errors for
-non-existent resources because they may have been deleted hierarchically.
+The uninstall deletes the RBAC permissions and all resources hierarchically
+under the `istio-system` namespace. It is safe to ignore errors for non-existent
+resources because they may have been deleted hierarchically.
 
 {{< text bash >}}
 $ istioctl manifest generate --set profile=demo | kubectl delete -f -
 {{< /text >}}
 
+The `istio-system` namespace is not removed for safety reasons. If you desire
```

Comment:
What safety reasons?

Reply:
Please clarify. Maybe something like this? Is this right?
```suggestion
The `istio-system` namespace is not removed so you can inspect the contents, in case the previous command didn't function as expected. If you desire
```

Reply:
It may contain other data from the users, beyond those that were installed by the installer. It's a good practice not to remove a namespace. If the user wants to do that, they still can do it, manually.

Reply:
Done, thanks.

Reply:
I see. In that case, I'd suggest simpler wording. See below.